### DISCUSSION TITLE
Agentless connect for RDP and VNC

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5891] [General]
Hi, we're looking for a remoteaccess solution. So i found meshcentral and the other additional tools. This looks great for us. 
But in order to run a testserver with meshcentral i want to know, if the clients (mostly windows) could be connected without an agent running on it?
For some clients we aren't allowed to install any software on the maschines, so the connection to this maschines could be with VNC or RDP. I need no monitoring for them, so no need for an agent!!
Some of the other maschines are regular PCs with Windows and there we could deploy the agent over out softwaredeployment.
So this is the nearly most important question for us. 
Grouping maschines and users should be configured as it should be regular in advanced software. 
Thanks and regards Sebastian

**Comment by si458** [comment]
You can get the end user to download the 'meshcentral assistant' which will allow u to control the remote computer with the users consent

Or

Download the agent and again ask the end user to click connect and consent (but using agent requires admin rights where as the assistant can connect to the current logged in user without admin rights)

Also if your meshcentral server can connect directly to the remote devices via ip addresses, u can enable local groups with web-vnc/web-rdp/web-ssh and then connect directly to the remote devices using the web ui

Does that help?

**Comment by simpsonetti** [comment]
Hi si458, yes, it helps . 
I'm trying to explain it more detailed: To connect to a device, could be done by using RDP (or VNC). In smallest configurationssetting this connect to a device could be done with the "jumphost" meshcentral by only logging into meshcentral and in dependency to the credentials it will be shown only the devices is is configured for the user. So after that the shown devices could be reached by clicking on it and using the credentials for this device. This will be done only by using the webinterface. 
And this without any (mesh-agent) installed on the remote system?!
This is quite the most important part by evaluating meshcentral. 

I see, i have to install meshcentral in a dev-environment and try it out.
Where can i enable these local groups?

**Comment by tschettervictor** [comment]
> Hi si458, yes, it helps . I'm trying to explain it more detailed: To connect to a device, could be done by using RDP (or VNC). In smallest configurationssetting this connect to a device could be done with the "jumphost" meshcentral by only logging into meshcentral and in dependency to the credentials it will be shown only the devices is is configured for the user. So after that the shown devices could be reached by clicking on it and using the credentials for this device. This will be done only by using the webinterface. And this without any (mesh-agent) installed on the remote system?! This is quite the most important part by evaluating meshcentral.
> 
> I see, i have to install meshcentral in a dev-environment and try it out. Where can i enable these local groups?

They are enabled by default. Just click “Add Group” and make sure you specify “Local Devices - No Agent”
